pkg/kubelet/status/status_manager.go

```diff
@@ -190,6 +190,12 @@ func (m *manager) SetPodStatus(pod *v1.Pod, status v1.PodStatus) {
 	m.updateStatusInternal(pod, status, pod.DeletionTimestamp != nil)
 }
 
+const (
+	unknown       = iota
+	container     = iota
+	initContainer = iota
```

Comment:
nit: why repeat the `iota` ?

Reply:
This is to assign unique values to the 3 identifiers.